### PR TITLE
Image pull policy checkbox

### PR DESCRIPTION
**Pull request by stuart-warren**
Add checkbox to select imagePullPolicy for container

### CODE REVIEW

**Comment by stuart-warren** [comment]
https://jenkins.ci.cloudbees.com/job/plugins/job/kubernetes-plugin/126/org.csanchez.jenkins.plugins$kubernetes/artifact/org.csanchez.jenkins.plugins/kubernetes/0.6-SNAPSHOT/kubernetes-0.6-SNAPSHOT.hpi

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java [hunk 1]
```diff
@@ -216,6 +216,7 @@ private String getIdForLabel(Label label) {
     private Pod getPodTemplate(KubernetesSlave slave, Label label) {
         final PodTemplate template = getTemplate(label);
         String id = getIdForLabel(label);
+        String shouldAlwaysPullImage = template.isAlwaysPullImage() ? "Always" : "IfNotPresent";
```
Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: I would call this `imagePullPolicy`

Comment (carlossg) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: actually I would get rid of the var and just use it below

`.withImagePullPolicy(template.isAlwaysPullImage() ? "Always" : "IfNotPresent")`

**Comment by carlossg** [comment]
Just added a minor comment, and it's good to go after rebasing

**Comment by stuart-warren** [comment]
Hopefully that's OK?

**Comment by carlossg** [comment]
there are 3 commits when there should be only one, your `checkbox to set imagePullPolicy`

**Comment by stuart-warren** [comment]
Sorted.

**Comment by carlossg** [comment]
thanks!

**Pull request merged**